integration_test/scenarios/email_detailed/deformed_inlined_image_scenario.dart

```diff
@@ -21,7 +21,7 @@ class DeformedInlinedImageScenario extends BaseTestScenario {
     <br>
     <img 
       src="https://example.com/image.jpg"
-      style="width: 2000px; height: 200px;" 
+      style="width: 2000px;height: 200px;" 
```

Comment:
What happens with the case of `style="width: 2000px; height: 200px;"`

Reply:
No problem, we just want to follow standard html style attributes. I added tests for these cases